lua/bufferline/buffer.lua

```diff
@@ -33,7 +33,7 @@ local utils = require'bufferline.utils'
 --- @alias bufferline.buffer.activity.name 'Inactive'|'Alternate'|'Visible'|'Current'
 
 --- The character used to delimit paths (e.g. `/` or `\`).
-local separator = package.config:sub(1,1)
+local separator = package.config:sub(1, 1)
```

Comment:
All the other function calls have spaces between parameters, so I'm just enforcing the style here.